integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+
+        z, x, y = (16, 34111, 23064)
+
+        self.generate_fixtures(
+            dsl.is_in('CH', z, x, y),
+            # https://www.openstreetmap.org/way/435454755
+            dsl.way(435454755, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'junction': u'roundabout',
+                'lanes': u'1',
+                'lit': u'yes',
+                'maxspeed': u'50',
+                'sidewalk': u'separate',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'is_in': u'CH',
+                'name': u'Hauptstrasse 1',
+                'name:de': u'Hauptstrasse 1',
+                'name:fr': u'Route principale 1',
+                'ref': u'1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[435454755]),
+            dsl.relation(2, {
+                'name': u'Hauptstrasse 10',
+                'network': u'ch:national',
+                'ref': u'10',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[435454755]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 435454755,
+                'network': u'CH:national',
+            })
+
+    def test_chnationalstrasse(self):
+        import dsl
+
+        z, x, y = (16, 34566, 23088)
+
+        self.generate_fixtures(
+            dsl.is_in('CH', z, x, y),
+            # https://www.openstreetmap.org/way/26686903
+            dsl.way(26686903, dsl.tile_diagonal(z, x, y), {
+                'bicycle': u'yes',
+                'cycleway': u'no',
+                'destination:backward': u'Davos',
+                'highway': u'primary',
+                'lanes': u'2',
+                'lanes:backward': u'1',
+                'maxspeed': u'80',
+                'oneway': u'no',
+                'ref': u'28',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'name': u'Hauptstrasse 28',
+                'ref': u'28',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[26686903]),
+            dsl.relation(2, {
+                'name': u'Ausbau N28 Prättigauerstrasse',
+                'network': u'ch:Nationalstrasse',
+                'operator': u'Schweizerische Eidgenossenschaft',
+                'ref': u'N28 (Projekt)',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[26686903]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 26686903,
+                'network': u'CH:Nationalstrasse',
```

Comment:
kinda odd that one is `CH:national` and the other is `CH:Nationalstrasse` but data is king?

Reply:
If we think they're the same thing, we can normalise them to the same network. I haven't (yet) looked in depth at Switzerland, so I'm not sure what their road network hierarchy looks like.

Do you think we should normalise them?

Reply:
We should generally speaking have "motorway" and "other national roads" network values Switzerland. 

Looks like in Switzerland the ways are in networks, but the network value is often just `motorway` for the RED ones ([example](https://www.openstreetmap.org/edit?way=271750328#map=17/46.97784/7.46862)) – while I expect `CH:motorway` to be exported.

- RED Number plate for motorway and highway (national and cantonal)
    - List of examples: https://de.wikipedia.org/wiki/Liste_der_Autobahnen_in_der_Schweiz
- BLUE Number plate Hauptstrasse (national, cantonal and municipal)
    - List of examples: https://de.wikipedia.org/wiki/Liste_der_Hauptstrassen_der_Schweiz

From the [occurrence counts on TagInfo](https://taginfo.openstreetmap.org/keys/network#values) it seems like `CH:Nationalstrasse` should be replaced with `CH:national`.

See also:

- https://de.wikipedia.org/wiki/Liste_der_Nationalstrassen_der_Schweiz
- https://de.wikipedia.org/wiki/Hauptstrasse
- https://de.wikipedia.org/wiki/Strassensystem_in_der_Schweiz_und_in_Liechtenstein